index.bs

```diff
@@ -264,12 +264,14 @@ infrastructure which allows those credentials to be used with {{CredentialsConta
 latter during [=Authentication=].
 
 Broadly, compliant [=authenticators=] protect [=public key credentials=], and interact with user agents to implement the
-[=Web Authentication API=]. Some authenticators might run on the same [=client device=] (e.g., smart phone, tablet, desktop PC) as
-the user agent is running on. For instance, such an authenticator might consist of a Trusted Execution Environment (TEE) applet,
-a Trusted Platform Module (TPM), or a Secure Element (SE) integrated into the [=client device=] in conjunction with some means
-for [=user verification=], along with appropriate driver software to mediate access to these components' functionality. Other
-authenticators might operate autonomously from the [=client device=] running the user agent, and be accessed over a transport such
-as Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field Communications (NFC).
+[=Web Authentication API=].
+Implementing compliant authenticators is possible in software executing
+(a) on a general-purpose computing device,
+(b) on an on-device Secure Execution Environment, Trusted Platform Module (TPM), or a Secure Element (SE), or
+(c) off device.
+Authenticators being implemented on device are called Platform Authenticators.
+Authenticators being implemented off device (Roaming Authenticators) can be accessed over a transport such
```

Comment:
s/Roaming Authenticators/[=roaming authenticators=]/ ?

Reply:
above fine by me if we are caplitalizing those terms consistently

Reply:
Looks like "roaming authenticator" is consistently lowercase.